src/components/layout/navbar/NavBar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import logo from './logo.svg';
+import { NavLink } from 'react-router-dom';
+import LoggedIn from './SignedInLink'
+import LoggedOut from './SignedOutLink'
+import { connect } from 'react-redux'
+
+const NavBar = (props) => {
+  const { auth } = props;
+  // const navlinks = auth.uid ? <LoggedIn /> : <LoggedOut />
+  return(
+    <nav className="nav-wrapper navy darken-2">
+       <div className="container">
+       <NavLink to="/" className="brand-logo">
+       <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" /> </header>
+       </NavLink>
+       <LoggedIn />
+       <LoggedOut />
+        {/* {navlinks} */}
```

Comment:
I think these need conditional rendering based on whether the user is logged in or not

Reply:
Done